service/src/main/java/com/github/swapbook/api/Users.java

```diff
@@ -0,0 +1,35 @@
+package com.github.swapbook.api;
+
+import com.github.swapbook.model.User;
+import com.github.swapbook.repositories.UsersRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class Users {
+
+    @Autowired
```

Comment:
Czy to jest taki automatyczny getter i setter ?

Reply:
nie do końca, przeczytaj to: 
https://stackoverflow.com/questions/3153546/how-does-autowiring-work-in-spring

bo mamy utworzony serwis UsersRepository i on jest jakby tylko jeden na całą aplikację. To bardziej jest taki konstuktor w tym miejscu, tylko nie tworzy kolejnej instancji klasy, tylko 
"wstrzykuje (inject)" tą klase.

Jest tutaj zaimplementowany wzorzec Singletona (https://refactoring.guru/design-patterns/singleton) i właśnie UsersRepository nim jest

Reply:
Ok czyli tak w javie się robi dependency injection ?
tak plus/minus